spec/requests/media/odk_response_spec.rb

```diff
@@ -207,4 +255,25 @@ def prepare_odk_media_upload_fixture(filename, form)
       formver: [form.number],
       itemcode: ODK::DecoratorFactory.decorate_collection(form.preordered_items).map(&:odk_code))
   end
+
+  def post_submission(submission_file, image_name, image, incomplete)
+    submission_params = {
+      xml_submission_file: submission_file,
+      image_name => image
+    }
+    submission_params["*isIncomplete*"] = "yes" if incomplete
+    post(submission_path, params: submission_params, headers: auth_header)
+  end
+
+  def expect_submission(status, num_response, num_attachments)
```

Comment:
for clarity/readability of the spec code above, can you please make these method params named? like `expect_submission(status: :created, num_response: 1, num_attachments: 2)`

Reply:
done